packages/styles/src/theme/index.ts

```diff
@@ -5,40 +5,27 @@ import '@fontsource/roboto'
 import muiDefaultProps from './muiDefaultProps'
 import getMuiThemeOverrides from './muiThemeOverrides'
 
-declare module '@material-ui/core/styles/createMuiTheme' {
-  export interface Theme {
-    layout: {
-      sideBarFullWidth: React.CSSProperties['width']
-      sideBarCollapsedWidth: React.CSSProperties['width']
-      navLinkTextWidth: React.CSSProperties['width']
-      toolBarWidth: React.CSSProperties['width']
-      navBarTop: React.CSSProperties['height']
-      navBarBottom: React.CSSProperties['height']
-      pageHeaderHeight: React.CSSProperties['height']
-      pagePadding: React.CSSProperties['padding']
-      toolContainerWidth: React.CSSProperties['width']
-      scrollBarWidth: React.CSSProperties['width']
-      tabHeight: React.CSSProperties['width']
-      paneDividerWidth: React.CSSProperties['width']
-    }
-  }
-  // allow configuration using `createMuiTheme`
-  export interface ThemeOptions {
-    layout?: {
-      sideBarFullWidth?: React.CSSProperties['width']
-      sideBarCollapsedWidth?: React.CSSProperties['width']
-      navLinkTextWidth: React.CSSProperties['width']
-      toolBarWidth: React.CSSProperties['width']
-      navBarTop: React.CSSProperties['height']
-      navBarBottom: React.CSSProperties['height']
-      pageHeaderHeight: React.CSSProperties['height']
-      pagePadding: React.CSSProperties['padding']
-      toolContainerWidth: React.CSSProperties['width']
-      scrollBarWidth: React.CSSProperties['width']
-      tabHeight: React.CSSProperties['width']
-      paneDividerWidth: React.CSSProperties['width']
-    }
-  }
+interface LayoutThemeProps {
+  sideBarFullWidth: React.CSSProperties['width']
+  sideBarCollapsedWidth: React.CSSProperties['width']
+  navLinkTextWidth: React.CSSProperties['width']
+  toolBarWidth: React.CSSProperties['width']
+  navBarTop: React.CSSProperties['height']
+  navBarBottom: React.CSSProperties['height']
+  pageHeaderHeight: React.CSSProperties['height']
+  pagePadding: React.CSSProperties['padding']
+  toolContainerWidth: React.CSSProperties['width']
+  scrollBarWidth: React.CSSProperties['width']
+  tabHeight: React.CSSProperties['width']
+  paneDividerWidth: React.CSSProperties['width']
+}
+
+export interface LumyTheme extends Theme {
+  layout?: LayoutThemeProps
+}
+
+export interface LumyThemeOptions extends ThemeOptions {
```

Comment:
It looks like LayoutThemeProps, LumyTheme, and LumyTheme here are a duplicate of what's already available in:
../theme/theme.ts
I suppose we could import it from there.

Reply:
Well spotted, removed!